src/main/java/chatterchicken/ChatterChicken.java

```diff
@@ -0,0 +1,74 @@
+import java.util.NoSuchElementException;
```

Comment:
`package chatterchicken;` is missing
```suggestion
package chatterchicken;

import java.util.NoSuchElementException;
```